### PR TITLE
Add repo field so npm stops complaining

### PR DESCRIPTION
**Pull request by duereg**
Add repo field so when you install write-good you don't get warnings from npm.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,10 @@
   "directories": {
     "test": "test"
   },
+  "repository": {
```
Comment (stevemao) [inline package.json]: or even simpler, just `"repository": "btford/weasel-words"`

**Comment by RichardLitt** [comment]
@btford I think this should just be merged in. The Travis check isn't relevant, and the code is correct. I don't have write access or I would pull it in now.

**Comment by btford** [comment]
Added you – feel free to merge!

**Comment by RichardLitt** [comment]
@btford Thanks! Done.

**Pull request merged**